### DISCUSSION TITLE
Improving Osu!Mania on mobile phones

### DISCUSSION THREAD

**Discussion by CharaYozora** [ppy/osu#25449] [Ideas]
With the addition of playable tiles (_which you just made it bigger i guess_) begs the question, **"Is it actually enough to make it competitive with the upcoming ranked gameplay you're introducing in the game?"**

I guess it's just me that's being weirded out but i have played some other rhythm games, but i do have a suggestion for mania to improve in lazer

**"What if we could make 4 seperate tiles for the ease of a competitive and actual playable Mania ruleset/mode?"**

Ok, hear me out lmao
What in suggesting here is like the game in roblox called _Funjy Friday_ where they have a toggleable setting called game tile key where they seperate the playable area (when in moble) by how many keys they're playing in, mainly in 4 parts of the screen

I think it can be really hard to implement especially with the crossplay in mind with you, the developers of this game/project, to have more people playing in the game the comfortability that comes with it.

I hope y'all can consider this suggestion for more comfortability and competitive gaming environment for us mobile players, that's all :>
Thank youuuuuuuuu

**Comment by ruwuuu** [comment]
why not keep the columns how they were before and just copy the takio ui to mania with slight changes
